src/components/header/connect.tsx

```diff
@@ -1,22 +1,28 @@
+'use client'
+
 import { useEffect } from 'react'
 
 import { useAccount } from 'wagmi'
 
 import { ConnectButton } from '@rainbow-me/rainbowkit'
 
-import { useNetwork } from '@/lib/hooks/useNetwork'
+import { useLedgerStatus } from '@/lib/hooks/use-ledger-status'
+import { useNetwork } from '@/lib/hooks/use-network'
 import { logConnect } from '@/lib/utils/api/analytics'
 
 export const Connect = () => {
   const { address } = useAccount()
   const { chainId } = useNetwork()
+  const { isRunningInLedgerLive } = useLedgerStatus()
```

Comment:
Is this used at all, or can we keep it out of the code for now?

Reply:
Will remove. 👍  Missed this during rebasing.